x-pack/legacy/plugins/code/public/components/commits/commit_link.tsx

```diff
@@ -14,10 +15,10 @@ interface Props {
 }
 
 export const CommitLink = ({ repoUri, commit, children }: Props) => {
-  // const href = DIFF.replace(':resource/:org/:repo', repoUri).replace(':commitId', commit);
+  const href = DIFF.replace(':resource/:org/:repo', repoUri).replace(':commitId', commit);
```

Comment:
This is basically the same as [this line](https://github.com/elastic/kibana/pull/46714/files#diff-e47dc454f8b3550350afd21ba3621aceR57) but with a totally different implementation, right? 😉

Reply:
This component is useless, I just delete this.